cluster/gce/gci/node-helper.sh

```diff
@@ -37,5 +37,6 @@ function get-node-instance-metadata-from-file {
 function create-linux-node-instance-template {
   local template_name="$1"
   ensure-gci-metadata-files
+  # shellcheck disable=2154 # 'scope_flags' is assigned by upstream
```

Comment:
Can you add a comment above this function that indicates that it expects `scope_flags` to be set? eg:
```
# Assumed vars:
#   scope_flags
# Parameters:
#   $1: template name (required).
function create-linux-node-instance-template {
```

Reply:
looks to be done now